src/modules/Account/account.controller.ts

```diff
@@ -122,12 +125,24 @@ export class AccountController {
     }
   }
 
+  @ApiTags('Password reset')
+  @Get('validatecode')
+  @RequirePermissions([Permissions['001']])
+  async validateCode(@Query() verifyCodeInput: VerifyCodeInput) {
```

Comment:
Atualmente você espera o código e o id do usuário pela URL. Sugiro a utilização do método _POST_ e o envio das informações pela propriedade _Body_

```suggestion
  async validateCode(@Body() verifyCodeInput: VerifyCodeInput) {
```